projects/onto-search/src/lib/onto-search-field/onto-search-field.component.ts

```diff
@@ -2,24 +2,50 @@ import {Component, EventEmitter, Input, OnInit, Output, TemplateRef} from '@angu
 import {Observable, of} from 'rxjs';
 import {SearchModel} from './models/search-model';
 
+
 @Component({
   selector: 'onto-search-field',
   templateUrl: './onto-search-field.component.html',
   styleUrls: ['./onto-search-field.component.scss'],
 })
 export class OntoSearchFieldComponent implements OnInit {
+  /**
+   * Custom template reference. Will override the template used in autocomplete dropdown.
+   */
   @Input()
   customTemplate?: TemplateRef<any>;
+
+  /**
+   * Preselected states list.
+   */
   @Input()
   private selectedList: any[];
+
+  /**
+   * Autocomplete response
```

Comment:
I don't like the comment. It is an input that is a response?
Something like `Observable of data that is provided as autocomplete options` makes more sense to me.